src/Initializer/time_stepping/LtsWeights/LtsWeights.cpp

```diff
@@ -309,17 +334,17 @@ LtsWeights::ComputeWiggleFactorResult LtsWeights::computeBestWiggleFactor() {
 
   const auto bestWiggleFactor = mapNumberOfClustersToBestWiggleFactor[minAdmissibleNumberOfClusters];
   const auto bestCostEstimate = mapNumberOfClustersToLowestCost[minAdmissibleNumberOfClusters];
-  logInfo(rank) << "The best wiggle factor after merging clusters is" << bestWiggleFactor
-                << "with cost" << bestCostEstimate;
+  logInfo(rank) << "The best wiggle factor is" << bestWiggleFactor << "with cost"
+                << bestCostEstimate << "and" << minAdmissibleNumberOfClusters << "time clusters";
 
-  logInfo(rank) << "Cost decreased" << (baselineCost - bestCostEstimate) / baselineCost * 100
-                << "% with absolute cost difference" << baselineCost - bestCostEstimate
-                << "compared to the default wiggle factor of" << maxWiggleFactor;
+  logInfo(rank) << "Cost decreased" << (*baselineCost - bestCostEstimate) / *baselineCost * 100
```

Comment:
the cost will be in most cases a cost increase, because the baseline is the minimum cost!
I would then describe it as "Due to clustering, cost increase"
(and remove Note: Cost increased due to cluster merging)

(or you can compare with the baseline wigglefactor=1 at this stage).